pkgs/misc/cups/default.nix

```diff
@@ -33,7 +33,9 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ pkgconfig removeReferencesTo ];
 
   buildInputs = [ zlib libjpeg libpng libtiff libusb gnutls libpaper ]
-    ++ optionals stdenv.isLinux [ avahi pam dbus systemd acl ]
+    ++ optionals stdenv.isLinux [ avahi pam dbus acl ]
+    # systemd can only be compiled when not using musl
+    ++ optional (stdenv.isLinux && !stdenv.hostPlatform.isMusl) systemd
```

Comment:
Same comment as https://github.com/NixOS/nixpkgs/pull/71563#discussion_r337311093

Reply:
This has been done in 3a4584c.